### ISSUE TITLE
last.fm: add the origin track as the first track in a generated playlist

### ISSUE THREAD

**Issue by gitgetgot** [hymerman/foo_bestversion#26]
Please add the origin track as the first track in the generated playlist.

So, when doing a search "Tracks similar to footrack", the track "footrack" is the first track in the generated playlist.

**Comment by hymerman** [comment]
Good idea! Should probably go in as an option though - which needs #3.
